app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ActiveRecord::Base
+  attr_accessor :remember_token
```

Comment:
__[reek]__
* [Attribute] is a writable attribute [:ref](https://github.com/troessner/reek/blob/master/docs/Attribute.md)